### ISSUE TITLE
24/7 snipe

### ISSUE THREAD

**Issue by petris11** [lopotras/WoltPartner-Schedule#14]
Hello, I want to ask if it would be possible to somehow simply rewrite the code so that the program refreshes 24/7 and checks free shifts 24/7? Thanks

**Comment by woolfakos1997** [comment]
I seek someone too if u find anyone mail me plz 🙏

**Comment by lopotras** [comment]
I don't think this would be much of a problem. I don't have access to the partner app anymore to even test, but can try to help out, if you're up to do it.

As to functionality how I'd approach it is continuously refresh and then check day by day each available day. On each day if there are more shifts that fit the screen, you'd have to scroll down until the bottom to make sure that all of them have been checked:
1. Refresh
2. Check availability for current day, for shifts showing up on screen
3. Scroll down
4. After scrolling down recheck the shifts for availability
5. (Could repeat point 3 & 4 if needed)
6. Move to next day
7. Repeat points 2-6 until last day
8. Return back to point 1.
